### DISCUSSION TITLE
Accessing the web console

### DISCUSSION THREAD

**Discussion by goneph1sh1ng** [Security-Onion-Solutions/securityonion#6400] [General]
Hey everyone, I've installed Security onion, now i'm trying to access the web console and it keeps coming back with the same error "website is busy, / not connected to the internet" any ideas?

**Comment by csanate** [comment]
Run so-status from the console to make sure all the services have started. It can take a few minutes for everything to boot up...

**Reply by goneph1sh1ng** [reply]
Hi there, sorry I should have mentioned in the original message that I've run so-allow. Still no joy unfortunately:/

**Comment by dougburks** [comment]
From https://github.com/Security-Onion-Solutions/securityonion/discussions/1720:

Please include the following details where you can:

- Security Onion version as seen in the lower left corner of SOC and in /etc/soversion. For example: 2.3.90
- Is this a cloud deployment or on-prem?
- Did you install from our Security Onion ISO image or did you perform a network installation?
- If network installation, did you install on CentOS 7 or Ubuntu?
- How many nodes do you have? How are each of those nodes configured? (ex. manager with 2 search nodes and 3 forward nodes)
- Are you experiencing issues monitoring network traffic? If so, are you sniffing from a tap or span port and what is the traffic volume?
- Does so-status show all services running?
- Do you get any failures when you run sudo salt-call state.highstate?
- Does the SOC Grid page show any failures?
- Explain your issue. For example: Installation fails when I select this series of options...
- Provide applicable logs. For example: if you are having problems during soup, provide /root/soup.log
